layouts/_default/baseof.html

```diff
@@ -10,6 +10,7 @@
 {{ partial "meta_json_ld.html" . }}
 {{ partial "scripts.html" . }}
 {{ partialCached "matomo.html" . }}
+{{ template "_internal/google_analytics.html" . }}
```

Comment:
This template doesn't work with Content security policy headers. I should add the solution I mention in  https://xdeb.org/post/2020/01/14/content-security-policy-headers-when-using-matomo-or-google-analytics/ to the theme.

Reply:
That's a good point, thanks!